src/UI/Component/ViewControl/Factory.php

```diff
@@ -94,4 +94,30 @@ public function section(Button $previous_action, \ILIAS\UI\Component\Component $
 	 */
 	public function sortation(array $options);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Data with a larger amount of entries might be displayed in parts
+	 *      by limiting the number of visible entries for a view.
+	 *      The pagination enables the user to select a range of entries.
+	 *      This control applies to all sorts of _structured_ data, like tables and lists.
+	 *   composition: >
+	 *      Pagination is a collection of shy-buttons.
+	 *   effect: >
+	 *      A click on an option will change the offset of the displayed data-list.
```

Comment:
What happens with the pagination itself? Which button will be active? What happens if the one on the left or right is clicked? Are left and right deactivated if there is no next?